.github/workflows/CI.yml

```diff
@@ -185,12 +185,9 @@ jobs:
           julia -e 'using Pkg; Pkg.add(PackageSpec(name="ReactionMechanismSimulator",rev="main")); using ReactionMechanismSimulator' || true 
 
       # non-regression testing
-      - name: Unit tests
-        run: make test-unittests
-      - name: Functional tests
-        run: make test-functional
-      - name: Database tests
-        run: make test-database
+      - name: Run Unit, Functional, and Database Tests
+        # aggregate into one command so we only have to eat the collection time once
+        run: make test-all
```

Comment:
Collection time on my machine is ~5mins, so maybe 5-10 minutes on GH actions? That x3 is tiresome

Also the database test is already called testDatabase or something like that, could we name the functional test along the same lines so that they can be run together but still labeled separately?